app/src/org/commcare/fragments/personalId/PersonalIdPhoneFragment.java

```diff
@@ -118,6 +124,27 @@ public void onDestroyView() {
         binding = null;
     }
 
+    private void checkGooglePlayServices() {
+        GoogleApiAvailability googleApiAvailability = GoogleApiAvailability.getInstance();
+        int status = googleApiAvailability.isGooglePlayServicesAvailable(requireActivity());
+        if (status != ConnectionResult.SUCCESS) {
+            String playServiceError = googleApiAvailability.getErrorString(status);
+            Logger.log(LogTypes.TYPE_MAINTENANCE,"Google Play Services issue:" + playServiceError);
+            Dialog dialog = googleApiAvailability.getErrorDialog(this, status, PLAY_SERVICES_RESOLUTION_REQUEST,
+                    dialog1 -> {
+                        onConfigurationFailure(playServiceError,
```

Comment:
instead of string error, pass `play_services_$status` as the failure code here and below.